README.md

```diff
@@ -169,7 +169,13 @@ The [`prettybat`](https://github.com/eth-p/bat-extras/blob/master/doc/prettybat.
 ### On Ubuntu
 *... and other Debian-based Linux distributions.*
 
-Download the latest `.deb` package from the [release page](https://github.com/sharkdp/bat/releases)
+You can install [the Ubuntu `bat` package](https://packages.ubuntu.com/eoan/bat) or [the Debian `bat` package](https://packages.debian.org/sid/bat) since Ubuntu Eoan 19.10 or Debian unstable sid.
+
+```bash
+apt install bat
+```
+
+On older releases, download the latest `.deb` package from the [release page](https://github.com/sharkdp/bat/releases)
```

Comment:
Maybe we could add: "If you want to run the latest release of `bat` or if you are on older versions of Ubuntu/Debian, you can download …"?

Reply:
Thanks. Done